Sources/StytchUI/AuthInputViewController.swift

```diff
@@ -85,28 +132,38 @@ final class AuthInputViewController: BaseViewController<AuthInputState, AuthInpu
             stackView.arrangedSubviews.map { $0.widthAnchor.constraint(equalTo: stackView.widthAnchor) }
         )
 
-        setupInputs()
+        setupPhoneNumberInput(input: phoneNumberInput)
+        setupPhoneNumberInput(input: whatsAppInput)
+        setupEmailInput(input: emailInput)
+
+        hideInputs(for: activeInput)
+    }
+
+    private func hideInputs(for input: Input) {
+        phoneNumberInput.isHidden = input == .email || input == .whatsapp
```

Comment:
Could these `||` be simplified to just `input != .phone` etc?

Reply:
I think that works! Will make that change before merging